docs/Rootless.md

```diff
@@ -5,7 +5,7 @@ layout: docs
 
 ### Context
 
-Practically all jailbreaks prior to iOS 15 have been 'rootful' as they work on and install files to the (root) system directories (e.g., `/usr`, `/Library`, `/Applications`, etc). Moving forward with iOS 15 and beyond, most if not all jailbreaks will have to be 'rootless' as Apple now prevents writing to the system directories with a protection called [Signed System Volume (SSV)](https://support.apple.com/guide/security/signed-system-volume-security-secd698747c9/web). There are workarounds for this, namely bindFS, but these are less than ideal.
+Practically all jailbreaks prior to iOS 15 have been 'rootful' as they work on and install files to the (root) system directories (e.g., `/usr`, `/Library`, `/Applications`, etc.). Moving forward with iOS 15 and beyond, most if not all jailbreaks will have to be 'rootless' as Apple now prevents writing to the system directories with a protection called [Signed System Volume (SSV)](https://support.apple.com/guide/security/signed-system-volume-security-secd698747c9/web). There are workarounds for this, namely bindFS, but these are less than ideal.
```

Comment:
> There are workarounds for this, namely bindFS, but these are less than ideal.

In my opinion, this seems out-of-scope for documentation on `rootless.h`. Bind mounts are used in a couple modern-day jailbreaks to achieve various effects, one of which is emulating a 'rootful' environment on iOS/iPadOS 15.0 and newer, where the root partition is sealed via SSV. 

The `rootless.h` documentation should not be the area that a developer is looking at to get information about how bind mounts are being used in modern jailbreaks. Therefore, I suggest that this be removed.